iphone/Classes/KrollBridge.m

```diff
@@ -770,7 +770,7 @@ -(id)loadCommonJSModule:(NSString*)code withSourceURL:(NSURL *)sourceURL
 	 *      "/Resources/views/login/window.js"  __dirname = "views/login"
 	 */
 
-	NSString *filename = [sourceURL lastPathComponent];
+	NSString *filename = [sourceURL lastPathComponent]; // FIXME This should be the absolute path to the file!
```

Comment:
Should be fixed with the mentioned PR. The line is only used to expose the `__filename` and that's done.